.github/workflows/sync_docs.yaml

```diff
@@ -0,0 +1,30 @@
+name: Sync docs from Discourse
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '53 0 * * *' # Daily at 00:53 UTC
+  # Triggered on push to branch "main" by .github/workflows/release.yaml
+  workflow_call:
+  pull_request:
+
+jobs:
+  sync-docs:
+    name: Open PR with docs changes
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Open PR with docs changes
+        uses: canonical/discourse-gatekeeper@main
```

Comment:
Please use the official one as here: https://github.com/canonical/spark-k8s-bundle/blob/main/.github/workflows/sync_docs.yaml

Reply:
Already using that I think?

Reply:
yes, I believe I raised the comment before the last commit! Resolving!